packages/server/lib/reporter.coffee

```diff
@@ -1,6 +1,8 @@
 _     = require("lodash")
 path  = require("path")
-Mocha = require("mocha")
+## mocha-v* is used to allow us to have later versions of mocha specified in devDependencies
+## and prevents accidently upgrading this one
+Mocha = require("mocha-v2")
```

Comment:
rename this to `mocha@2.4.5`

Reply:
had to use `mocha-2.4.5` since yarn will not allow the `@` symbol